src/organisms/home/Venue.vue

```diff
@@ -1,7 +1,79 @@
 <template>
-  <div>
+  <div id="venue-background">
     <div class="container">
-      Venue
+      <PageHeading :display="5">Location</PageHeading>
+      <div class="location">
+        <SubPageHeading><img
+            alt="🇸🇰"
+            src="https://s.w.org/images/core/emoji/11/svg/1f1f8-1f1f0.svg"
+          /><b>Kosice, Slovakia</b></SubPageHeading>
+      </div>
+      <SubPageHeading>
+        <b>Venue:</b> Kunsthalle/Hala umenia Košice
+      </SubPageHeading>
+      <MapViewer
+        :position="getPosition"
+        :zoom="17"
+        :enablePicker="true"
+        :fixed="false"
+        :tooltip="{
+          title: 'Kunsthalle Košice',
+          description: 'Rumanova 1, 040 01 Košice, Slovakia'
+        }"
+        class="map-viewer"
+      />
     </div>
   </div>
 </template>
+
+<script>
+import PageHeading from "@/atoms/PageHeading.vue";
+import SubPageHeading from "@/atoms/SubPageHeading.vue";
+import MapViewer from "@/atoms/MapViewer.vue";
+
+export default {
+  name: "VenueSection",
+
+  components: {
+    PageHeading,
+    SubPageHeading,
+    MapViewer
+  },
+
+  computed: {
+    getPosition: function() {
```

Comment:
I would treat it as a variable name, not method name.

Reply:
Done